git-helper.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
```

Comment:
We should add `set -ex` for easier debugging; https://www.peterbe.com/plog/set-ex

`-e` fails fast
`-x` prints out commands